e2e/protractor.conf.js

```diff
@@ -13,6 +13,10 @@ exports.config = {
   capabilities: {
     browserName: 'chrome',
   },
+  chromeDriver:
+    process.platform === 'win32'
+      ? '../node_modules/webdriver-manager/selenium/chromedriver_89.0.4389.114.exe'
+      : '../node_modules/webdriver-manager/selenium/chromedriver_89.0.4389.114',
```

Comment:
Implemented as a temporary workaround due to version mismatch of chromedriver in E2E test.